### DISCUSSION TITLE
Is it possible to install multiple Yellow in subdirectories?

### DISCUSSION THREAD

**Discussion by bulubebek** [datenstrom/community#666] [Ask a question] [answered]
Hello 👋

Is it possible to install yellow on multiple subdirectories in a single domain? example:

```
site.tld/user1
site.tld/user2
site.tld/user3
```

My goal is to host yellow for my friends, so they can have personal profiles/minisite under one domain.

I believe it relies on .htaccess, but I can't make it works after playing around with yellow's .htaccess. 😄
 
Thank you in advanced, and keep up your good works!

**Comment by schulle4u** [comment, marked as answer]
Yes, this is possible. Just copy the Yellow installation files in each subdirectory, including `.htaccess` and `yellow.php`. Depending on what is in your root directory you might have to fiddle a bit with existing htaccess roules. Some use cases are covered in [the troubleshooting guide](https://datenstrom.se/yellow/help/troubleshooting#problems-with-apache).

**Reply by bulubebek** [reply]
ah, thank you! I will give a test and report its result here.

**Reply by markseuffert** [reply]
I do this all the time for testing. Installations in subdirectories are absolutely independent from each other. Installations do not share resources. Since Datenstrom Yellow is relatively small in size it's not a problem to have multiple installations on your web server.
